evm/src/util.rs

```diff
@@ -40,3 +41,29 @@ pub fn trace_rows_to_poly_values<F: Field, const COLUMNS: usize>(
         .map(|column| PolynomialValues::new(column))
         .collect()
 }
+
+/// Returns the 32-bit little-endian limbs of a `U256`.
+pub(crate) fn u256_limbs<F: Field>(u256: U256) -> [F; 8] {
+    u256.0
+        .into_iter()
+        .flat_map(|limb_64| {
+            let lo = (limb_64 & 0xFFFFFFFF) as u32;
```

Comment:
nit: Isn't that the same as `limb_64 as u32`? Or do you think it's clearer that way?

Reply:
Ah, somehow I thought it would fail in debug mode, but looks like that's not the case.